.gitmodules

```diff
@@ -1,6 +1,3 @@
 [submodule "vendor/rdma-core"]
-	path = vendor/rdma-core
-	url = https://github.com/linux-rdma/rdma-core.git
-[submodule "vendor/bindgen"]
-	path = vendor/bindgen
-	url = https://github.com/servo/rust-bindgen.git
+	path = ibverbs-sys/vendor/rdma-core
+	url = https://github.com/rdelfin/rdma-core.git
```

Comment:
Hmm, why did you switch which repository this is using? I'd prefer to continue using the official one.

Reply:
This was mostly since it seems like they have compiler errors in clang (or at least, in directories that are nested enough) but I'll see if there's a better solution

Reply:
You could try sending the patches upstream? Always good if we can help the greater ecosystem!